src/PresentationalComponents/Modals/DisableRule.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Modal, Button, Form, FormGroup, TextInput, Checkbox } from '@patternfly/react-core';
+import { injectIntl } from 'react-intl';
+import { connect } from 'react-redux';
+import { addNotification } from '@redhat-cloud-services/frontend-components-notifications';
+
+import API from '../../Utilities/Api';
+import { BASE_URL } from '../../AppConstants';
+import messages from '../../Messages';
+
+const DisableRule = ({ handleModalToggle, intl, isModalOpen, displaySingleSystemCheckbox, rule, afterDisableFn, addNotification }) => {
+    const [justification, setJustificaton] = useState('');
+    const [singleSystem, setSingleSystem] = useState(false);
+
+    const disableRule = async () => {
+        try {
+            if (rule.reports_shown) {
+                // eslint-disable-next-line camelcase
+                await API.post(`${BASE_URL}/ack/`, { rule_id: rule.rule_id, justification });
```

Comment:
I am just curious. Is the trailing `/` required in the URL?

Reply:
yeah, otherwise we get a 301 :-/